src/cf-core/src/cf-base.less

```diff
@@ -548,13 +538,13 @@ select[multiple] {
     @font-size: 16px;
 
     display: inline-block;
-    margin: 0;
-    padding: unit(6 / @base-font-size-px, em);
-    font-family: Arial, sans-serif;
-    font-size: unit(@font-size / @base-font-size-px, em);
-    background: @input-bg;
+    padding: unit( 6 / @base-font-size-px, em );
     border: 1px solid @input-border;
     border-radius: 0;
+    margin: 0;
+    background: @input-bg;
+    font-family: Arial, sans-serif;
```

Comment:
I think we intentionally wanted to avoid using the webfont in form fields? The Design Manual [doesn't explicitly say](http://cfpb.github.io/design-manual/page-components/form-fields.html), but the examples all show Arial.